single-template.php

```diff
@@ -46,45 +46,19 @@
                     <!-- Requires update block -->
                     <section class="cell small-12 update-needed-notification"
                              style="display: <?php echo esc_html( ( isset( $dt_post['requires_update'] ) && $dt_post['requires_update'] === true ) ? "block" : "none" ) ?> ">
-                        <div class="bordered-box detail-notification-box" style="background-color:#F43636">
-                            <h4>
-                                <img src="<?php echo esc_html( get_template_directory_uri() . '/dt-assets/images/alert-circle-exc.svg' ) ?>"/>
-                                <?php echo esc_html( sprintf( __( 'This %s needs an update.', 'disciple_tools' ), strtolower( $post_settings["label_singular"] ) ) ) ?>
-                            </h4>
-                            <p><?php esc_html_e( 'Please provide an update by posting a comment.', 'disciple_tools' )?></p>
-                        </div>
+                        <a href="#comment-activity-section">
```

Comment:
@micahmills Brilliant!
this one is a bit weird on desktop, is just scrolls the page down some:

Reply:
![image](https://user-images.githubusercontent.com/24901539/106268496-7fab9880-622b-11eb-84af-1e185b053550.png)

Reply:
Yeah that would be a bit weird because presumably the comments section is on the screen in the second column on the desktop view. Let me think about this and I am sure we can come up with a way to handle it.